src/naksu/boxversion/boxversion.go

```diff
@@ -0,0 +1,202 @@
+    boxVersion = vagrantBoxAvailVersionDetailsCache.boxVersion
+	}
+
+  mebroutines.LogDebug(fmt.Sprintf("GetVagrantBoxAvailVersionDetails returns: [%s] [%s]", boxString, boxVersion))
+  return boxString, boxVersion, boxError
+}
+
+// reallyGetVagrantBoxAvailVersionDetails returns version string (e.g. "digabi/ktp-qa") and
+// version number (e.g. "69") by getting Vagrantfile -> metadata.json
+func reallyGetVagrantBoxAvailVersionDetails () (string, string, error) {
+	// Phase 1: Get Abitti Vagrantfile
+	strVagrantfile, errVagrantfile := network.DownloadString(constants.AbittiVagrantURL)
+	if errVagrantfile != nil {
+		mebroutines.LogDebug(fmt.Sprintf("Could not download Abitti Vagrantfile from %s", constants.AbittiVagrantURL))
+		return "", "", errors.New("could not download abitti vagrantfile")
+	}
+
+	reBoxString := regexp.MustCompile(`config.vm.box = "(.+)"`)
+	reMetadata := regexp.MustCompile(`config.vm.box_url = "(.+)"`)
+
+	boxStringMatches := reBoxString.FindStringSubmatch(strVagrantfile)
+	boxMetadataMatches := reMetadata.FindStringSubmatch(strVagrantfile)
+
+	if len(boxStringMatches) != 2 {
+		mebroutines.LogDebug("Could not find config.vm.box from Abitti Vagrantfile:")
+		mebroutines.LogDebug(strVagrantfile)
+		return "", "", errors.New("could not find config.vm.box")
+	}
+
+	if len(boxMetadataMatches) != 2 {
+		mebroutines.LogDebug("Could not find config.vm.box_url from Abitti Vagrantfile:")
+		mebroutines.LogDebug(strVagrantfile)
+		return "", "", errors.New("could not find config.vm.box_url")
+	}
+
+	// Phase 2: Get vagrant metadata.json
+	strMetadata, errMetadata := network.DownloadString(boxMetadataMatches[1])
+	if errMetadata != nil {
+		mebroutines.LogDebug(fmt.Sprintf("Could not download Abitti metadata from %s", boxMetadataMatches[1]))
+		return "", "", errors.New("could not download abitti metadata")
+	}
+
+	reVersion := regexp.MustCompile(`"version": "(\d+)"`)
+
+	versionMatches := reVersion.FindStringSubmatch(strMetadata)
+
+	if len(versionMatches) != 2 {
+		mebroutines.LogDebug("Could not find version number from Vagrant metadata:")
+		mebroutines.LogDebug(strMetadata)
+		return "", "", errors.New("could not find version number from vagrant metadata")
+	}
+
+	return boxStringMatches[1], versionMatches[1], nil
+}
+
+// GetVagrantBoxType returns the type string (Abitti server or Matric Exam server) for vagrant box name
+func GetVagrantBoxType(name string) string {
+	if GetVagrantBoxTypeIsAbitti(name) {
+		return xlate.Get("Abitti server")
+	}
+
+	if GetVagrantBoxTypeIsMatricExam(name) {
+		return xlate.Get("Matric Exam server")
+	}
+
+
+	return "-"
+}
+
+// GetVagrantBoxTypeIsAbitti returns true if given box name string
+// belongs to an Abitti vagrant box
+func GetVagrantBoxTypeIsAbitti(name string) bool {
+	return (name == "digabi/ktp-qa")
+}
+
+// GetVagrantBoxTypeIsMatricExam returns true if given box name string
+// belongs to a Matriculation Examination vagrant box
+func GetVagrantBoxTypeIsMatricExam(name string) bool {
```

Comment:
I'd use full `Matriculation` word here in naming

Reply:
Renamed